packages/ai-jsx/src/core/opentelemetry.ts

```diff
@@ -91,13 +100,11 @@ export function openTelemetryStreamRenderer(streamRenderer: StreamRenderer): Str
         // Nothing is currently tracing this memoized subtree.
         startup = (span) => memoizedIdsToActiveSpanContexts.set(memoizedId, span.spanContext());
         cleanup = () => memoizedIdsToActiveSpanContexts.delete(memoizedId);
-        const newContext = activeContext.setValue(activeMemoizedIdStorage, { id: memoizedId, isOwner: true });
-        return context.with(newContext, () =>
-          tracer.startActiveSpan(
-            `<${renderable.tag.name}>`,
-            { attributes: spanAttributesForElement(renderable) },
-            renderInSpan
-          )
+        return tracer.startActiveSpan(
+          `<${renderable.tag.name}>`,
+          { attributes: spanAttributesForElement(renderable) },
+          activeContext.setValue(activeMemoizedIdStorage, { id: memoizedId, isOwner: true }),
+          renderInSpan
```

Comment:
What's the significance of passing `newContext` as a param to `startActiveSpan` instead of doing `context.with`?